### PR TITLE
ci: add configs-periodics.yaml

### PR DESCRIPTION
**Pull request by hi-rustin**
close https://github.com/ti-community-infra/tichi/issues/317

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: prow/jobs/ti-community-infra/configs/configs-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+periodics:
+  - name: periodic-configs-autobump
+    # Run at minute 5 past every hour from 18 through 23 on every Monday (At UTC+8 timezone).
+    cron: "05 10-15 * * 1"
+    decorate: true
+    extra_refs:
+      # Check out the repo containing the config and deployment files for your Prow instance.
+      - org: ti-community-infra
+        repo: configs
+        base_ref: master
```
Comment (Mini256) [inline prow/jobs/ti-community-infra/configs/configs-periodics.yaml:10]:
```suggestion
        base_ref: main
```

Comment (hi-rustin) [reply prow/jobs/ti-community-infra/configs/configs-periodics.yaml:10]: Good catch!

**Comment by hi-rustin** [comment]
Thanks!

**Pull request merged**